be/src/controllers/rest.router.ts

```diff
@@ -65,13 +66,20 @@ router.put('/vocab/:id', (req: Request, res: Response, next: NextFunction) => {
 
 router.get('/vocab/:id', (req: Request, res: Response, next: NextFunction) => {
   vocabularyService.getVocabular(req.params.id)
-    .then(v => vocabDbo2Dto(v))
-    .then(v => res.json(v))
-    .catch(next);
+      .then(v => vocabDbo2Dto(v))
+      .then(v => res.json(v))
+      .catch(next);
 });
 
-router.delete('/vocab/:id', (req: Request, res: Response, next: NextFunction) => {
-  next(new KnowledgeError(501, 'Not Implemented', 'DELETE /vocab/:id is not implemented'));
+router.delete('/vocab/:id', async (req: Request, res: Response, next: NextFunction) => {
+  const header = req.header('if-unmodified-since')
+  const date: Date = new Date(header)
+
+  vocabularyService.deleteVocab(req.params.id, date).then(result => {
+    if (result) {
+      res.status(204).end();
+    }
+  }).catch(next);
```

Comment:
Why `next`? Is there a global error handler?

Reply:
Take a look on #20. @ManuelTS implemented error handling for express.